book/website/collaboration/new-community/new-community-guide.md

```diff
@@ -0,0 +1,90 @@
+# Guide to Planning a Community
+
+What if you started a project with a thought like, “I have this great idea that I want to try on this public data!”?
+There is nothing to worry about if you’re the only one working on it.
+However, if you want to develop this project - you become responsible for making people feel included in your project.
+
+As a 'project lead', you want to set up a welcoming and inclusive environment and create the first set of visions and goals for your collaborators.
+You can not assume that everyone you collaborate with knows what is expected of them when they start to work with others on your project.
+Therefore it’s important to set the right expectations from the beginning for your community, even though you might not have planned on having one (see more details: {cite}`Sharan2020Apr`).
+
+## A checklist for planning collaboration in your project
+
+The checklist below will help you in making the process of establishing collaboration in your research project thoughtfully in a structured manner.
+
+The practices listed here are derived from and limited by the experiences of the authors who participate in several successful Open Research communities and lead community-driven projects such as [The Carpentries](https://carpentries.org), [Mozilla Open Leaders](https://mozilla.github.io/open-leadership-training-series/), [Open Life Science](openlifesci.org/) and _The Turing Way_.
+While reading this chapter, please be aware that you may need to make adjustments for the projects that may be very different in nature (for example, not entirely open source).
+
+### 1. Choose a communication platform
+
+- When leading an open project, use collaborative and open platforms such as [GitHub](http://github.com/).
+- Evaluate the need for any real-time communications such as if a text chat system like [Slack](https://slack.com) or [Riot/Matrix](https://about.riot.im/free) will be useful or a mailing list be sufficient (read details {ref}`Communication Channels <cm-os-comms-channels>`).
+  - Consider a separate internal communication platform for your community members and an external one for showing what you’ve done to the rest of the world.
+- A [Twitter account](twitter.com/) or a simple website (such as on [Git pages](https://pages.github.com/)) can be useful external platforms.
+- Make sure that the choices of these platforms are made to ensure that there is a low barrier to join them.
+
+### 2. Provide a project summary file:
+
+- The first document in your project should be a project summary file, which on GitHub repository will be a [README.md file](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes).
+- This will provide basic information about your project so that people can evaluate why your project will be interesting for them.
+  - Here is [a template](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2) by the GitHub user [PurpleBooth](https://gist.github.com/PurpleBooth).
```

Comment:
I can't ever get that gist to load (do other people have the same problem?). If it's just on my end, then please ignore this. Otherwise, should it point to the GitHub repo template instead (instead of the gist?): https://github.com/PurpleBooth/a-good-readme-template, e.g.:
```suggestion
  - Here is [a template](https://github.com/PurpleBooth/a-good-readme-template) by the GitHub user [PurpleBooth](https://github.com/PurpleBooth).
```